.github/workflows/playwright.yml

```diff
@@ -0,0 +1,25 @@
+name: Playwright Tests
+on:
+  push:
+  pull_request:jobs:
```

Comment:
Run tests pushes and pulls from all branches, that way if we are using a feature branch and are making PR's to it, we can still have the benefit of these tests passing.